sldprt_diff.sh

```diff
@@ -14,7 +10,9 @@ fi
 diff_filename="$(echo "$1" | sed 's/\(.*\)\.\([^.]*\)/\1_OTHER\.\2/')"
 cp $2 $diff_filename
 
```

Comment:
```suggestion

echo "Opening SolidWorks to diff $left_filename and $right_filename..."
echo "SolidWorks must exit fully before this diff can exit"
```
same as the other comment

Reply:
edited to match new commits in `master`